tasks/system.yml

```diff
@@ -14,7 +14,7 @@
     undetermined_actions:
       - "{{default_notification.notification_method_id}}"
   with_items:
-    - { name: "Host Status", description: "Alarms when the specified host is down or not reachable", severity: "HIGH", expression: "host_alive_status > 0", match_by: ["target_host", "hostname"]}
```

Comment:
target_host was replaced by hostname, hostname was replaced by observer_host. Should we instead add observer_host? The way this is now is there is one alarm with potentially multiple observers. I can see reasons for both